docs/_data/table/en/user-configuration.yml

```diff
@@ -1,9 +1,8 @@
 alluxio.user.block.master.client.threads:
   The number of threads used by a block master client pool to talk to the block master.
 alluxio.user.block.worker.client.threads:
-  The number of threads used by a block worker client pool to talk to a remote block worker. It
-  determines the maximum number of Thrift connections between an Alluxio client and an Alluxio
-  worker.
+  The number of threads used by a block worker client pool for heartbeating to a worker. Increase
+  this value if worker failures affect client connections to healthy workers.
```

Comment:
Could you comment on what kind of worker failures may affect the client connections?